sdk/instrumentation/net/http/handler.go

```diff
@@ -45,16 +57,39 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		span.SetAttribute(key, value)
 	}
 
-	span.SetAttribute("http.url", r.URL.String())
+	url := r.URL.String()
+	span.SetAttribute("http.url", url)
+
+	// run url filters
+	for _, urlEvaluator := range h.urlFilters {
+		if urlEvaluator(url) {
+			w.WriteHeader(http.StatusForbidden)
+			w.Write([]byte(fmt.Sprintf("403 - Blocked by URL filter")))
```

Comment:
do we also write the body in the other agents when we block?

Reply:
In envoy we do. I'll just get rid of the body.